### ISSUE TITLE
what is the bert-score stand for?

### ISSUE THREAD

**Issue by linhaow** [L-Zhe/BTmPG#3]
When I run the bert-score script, I can get precision, recall and f score, what is the bert score in paper stand for

**Comment by linhaow** [comment]
Besides, there is re-scale option, have you use it ?

**Comment by L-Zhe** [comment]
Yes, I use the f-score as the final bert-score and leverage --rescale_with_baseline to make results more intuitive.
All these settings are explain in our paper "Towards Document-Level Paraphrase Generation with Sentence Rewriting and Reordering". You can get more details in it.


Lin.
________________________________
